src/BuildCheck.UnitTests/EndToEndTests.cs

```diff
@@ -105,6 +105,36 @@ public void SampleAnalyzerIntegrationTest_ReplayBinaryLogOfAnalyzedBuild(bool bu
         }
     }
 
+    [Theory]
+    [InlineData("warning", "warning BC0101")]
+    [InlineData("error", "error BC0101")]
+    [InlineData("suggestion", "BC0101")]
```

Comment:
is there a way to distinguish suggestion from the others?
If not by presence of specific text, than maybe via absence of specific texts (e.g. `{ "warning BC0101", "error BC0101" }`)?

Reply:
Thank you for the suggestion, update the test case 
Duplication removed :)